app/views/choose_an_option.scala.html

```diff
@@ -15,58 +15,54 @@
-                    <p class="paragraph-info ">@messages("file.size.info")</p>
-                    <p class="paragraph-info no-bottom-margin">@messages("processing.file")</p>
+                    <p class="paragraph-info govuk-body"><strong class="task-completed" id="file-processing">@messages("file.processing")</strong>@messages("file.upload.time", formattedDate.get)</p>
+                    <p class="paragraph-info govuk-body">@messages("file.size.info")</p>
+                    <p class="paragraph-info govuk-body govuk-!-margin-bottom-0">@messages("processing.file")</p>
                 </div>
             }
 
             @if(fileStatus==Ready){
                 <div class="task-list-item">
-                    <p><strong class="task-completed" id="download-result">@messages("file.ready")</strong>
-                        <a class="task-name" id="download-result-link" href="@routes.ChooseAnOptionController.renderUploadResultsPage" data-journey-click="link - click:Choose option to get residency status:Download your results">@messages("download.results")</a></p>
-                    <p class="paragraph-info no-bottom-margin small-margin-top">@messages("result.timescale", formattedDate.get)</p>
+                    <p><strong class="task-completed govuk-body" id="download-result">@messages("file.ready")</strong>
+                        <a class="task-name govuk-link govuk-body" id="download-result-link" href="@routes.ChooseAnOptionController.renderUploadResultsPage" data-journey-click="link - click:Choose option to get residency status:Download your results">@messages("download.results")</a></p>
+                    <p class="paragraph-info govuk-body govuk-!-margin-bottom-0 govuk-!-margin-top-4">@messages("result.timescale", formattedDate.get)</p>
                 </div>
             }
 
-            @if(fileStatus==UploadError || fileStatus==TimeExpiryError ){
+            @if(fileStatus==UploadError || fileStatus==TimeExpiryError){
                 <div class="task-list-item">
-                    <p class="paragraph info"><strong class="task-completed" id="file-problem">@messages("file.problem")</strong>@messages("problem.getting.result")</p>
-                    <p class="paragraph-info no-bottom-margin small-margin-top">
-                        @Html(
-                        messages(
-                        "file.problem.paragraph",
-                        "<a id='file-problem-link' class='file-problem-link' href='" + routes.FileUploadController.get() + "' data-journey-click='link - click:Choose option to get residency status:Upload a file'>" + messages("upload.file.again") + "</a>"
-                        ))
+                    <p class="paragraph info govuk-body"><strong class="task-completed" id="file-problem">@messages("file.problem")</strong>@messages("problem.getting.result")</p>
```

Comment:
Possible typo (looks like its been there a while), the class "paragraph info .... " seems like it is supposed to be "paragraph-info" like on line 62 below? Havent done any looking into it yet.